### PR TITLE
Update travis to PostgreSQL 12

### PR DESCRIPTION
**Pull request by pauloxnet**
Fix first point of #1031

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [flake8]
 exclude = djangodocs,.tox
-ignore = F405,W504
+ignore = F405,W292,W504
```
Comment (adamchainz) [inline setup.cfg:3]: This is "No newline at end of file" which I don't think we want to disable. What context did it appear in?

Comment (pauloxnet) [reply setup.cfg:3]: I know it's strange but without it checks are not passing.

Comment (adamchainz) [reply setup.cfg:3]: Which files? Can you fix the files instead?

Comment (pauloxnet) [reply setup.cfg]: I rebased and found it, the newline at the of `djangoproject/settings/dev.py` was missing.

The issue was added in this commit, and its strange that was merged as is
https://github.com/django/djangoproject.com/commit/46aa515ec852c33b3ef72319d98ed8e49a07c7e4#diff-87bbf741c56455e78767d6fd1b1f346c

Now all checks in my PR pass. Thanks.

**Review by carltongibson — COMMENTED** [review]
OK, so this looks fine... but we have Ops considerations.

Let me run to the broom-cupboard and find out more...

**Comment by adamchainz** [comment]
(It'd probably be good to follow up this with a move to GitHub Actions - happy to do that if you want.)

**Comment by carltongibson** [comment]
Yes, please @adamchainz! 💪

**Comment by pauloxnet** [comment]
> OK, so this looks fine... but we have Ops considerations.

Ok, ping me, if you need some feedback with the PostgreSQL 10 to 12 migration process.

**Comment by felixxm** [comment]
Superseded by #1151.

**Pull request closed without merging**